tests/TestData/TestTools/RouteVisitor/MiddlewareTestVisitor.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace CubaDevOps\Flexi\Test\TestData\TestTools\RouteVisitor;
```

Comment:
This class is not necessary if a test double is implemented on the Route/Router class to see if a method has been invoked through a control variable as in the example I left you in another comment.